app/models/sign_in/terms_code_container.rb

```diff
@@ -8,7 +8,8 @@ class TermsCodeContainer < Common::RedisStore
 
     attribute :code, String
     attribute :user_account_uuid, String
+    attribute :user_verification_id, String
```

Comment:
we _could_ just use `user_verification_id` here but doesn't really hurt to keep `user_account_uuid` and i don't want to change too much rn